docs/charts.md

```diff
@@ -25,6 +25,7 @@ wordpress/
   Chart.yaml        # A YAML file containing information about the chart
   LICENSE           # OPTIONAL: A plain text file containing the license for the chart
   README.md         # OPTIONAL: A human-readable README file
+  NOTES.txt         # OPTIONAL: A plain text file containing short usage notes
```

Comment:
How would you feel about putting this in `templates/NOTES.txt`? Since it is a template, it seems like it's not necessarily designed to stand on its own. And if we put it there, we don't have to special-case one file in the template renderer.